sdk/ai/Azure.AI.Projects/README.md

```diff
@@ -314,6 +315,91 @@ var attachment = new MessageAttachment(
 );
 ```
 
+#### Create Agent with Azure AI Search
+
+Azure AI Search is an enterprise search system for high-performance applications.
+It integrates with Azure OpenAI Service and Azure Machine Learning, offering advanced
+search technologies like vector search and full-text search. Ideal for knowledge base
+insights, information discovery, and automation. Creating an Agent with Azure AI
+Search requires an existing Azure AI Search Index. For more information and setup
+guides, see [Azure AI Search Tool Guide](https://learn.microsoft.com/azure/ai-services/agents/how-to/tools/azure-ai-search?tabs=azurecli%2Cpython&pivots=overview-azure-ai-search).
+
```

Comment:
Why is there "python" in the URL? I think you can remove all the query strings.

Reply:
Fixed.